test/scripts/e2e_subs/tealprogs/abi-min_balance.json

```diff
@@ -0,0 +1,69 @@
+{
```

Comment:
`AtomicABI` needs the ABI contract in order to dynamically create the caller metnods.